docs/user/installation_and_configuration.rst

```diff
@@ -301,6 +294,105 @@ In this case you may download SLF4J from http://www.slf4j.org/download.html. Ext
+    $ cd geomesa-kafka-$KAFKAVERSION_2.11-$VERSION
+    $ ls
+    bin/  conf/  dist/  docs/  examples/  lib/  LICENSE.txt
+
+The instructions below assume that the ``geomesa-kafka-$KAFKAVERSION_2.11-$VERSION`` directory is kept in the
+``geomesa-kafka/geomesa-kafka-dist/geomesa-kafka-$KAFKAVERSION-dist/target/`` directory, but the tools distribution may be moved elsewhere
+as desired.
+
+.. note::
+
+    You can configure environment variables and classpath settings in geomesa-kafka-$KAFKAVERSION_2.11-$VERSION/bin/geomesa-env.sh.
+
+In the ``geomesa-kafka-$KAFKAVERSION_2.11-$VERSION`` directory, run ``bin/geomesa configure`` to set up the tools.
+
+.. code-block:: bash
+
+    ### in geomesa-kafka-$KAFKAVERSION_2.11-$VERSION:
+    $ bin/geomesa-kafka configure
+    Using GEOMESA_KAFKA_HOME as set: /path/to/geomesa-kafka-$KAFKAVERSION_2.11-$VERSION
+    Is this intentional? Y\n y
+    Warning: GEOMESA_LIB already set, probably by a prior configuration.
+    Current value is /path/to/geomesa-kafka-$KAFKAVERSION_2.11-$VERSION/lib.
+
+    Is this intentional? Y\n y
+
+    To persist the configuration please update your bashrc file to include:
+    export GEOMESA_KAFKA_HOME=/path/to/geomesa-kafka-$KAFKAVERSION_2.11-$VERSION
+    export PATH=${GEOMESA_KAFKA_HOME}/bin:$PATH
+
+Update and re-source your ``~/.bashrc`` file to include the ``$GEOMESA_KAFKA_HOME`` and ``$PATH`` updates.
+
+.. warning::
+
+    Please note that the ``$GEOMESA_KAFKA_HOME`` variable points to the location of the ``geomesa-kafka-$KAFKAVERSION_2.11-$VERSION``
+    directory, not the main geomesa binary distribution directory!
+
+.. note::
+
+    ``geomesa-kafka`` will read the ``$GEOMESA_EXTRA_CLASSPATHS`` environment variable to load any
+    additional jars into the classpath. Use the ``geomesa classpath`` command in order to see what
+    JARs are being used.
+
+Due to licensing restrictions, dependencies for shape file support and raster
+ingest must be separately installed. Do this with the following commands:
+
+.. code-block:: bash
+
+    $ bin/install-jai.sh
+    $ bin/install-jline.sh
+
+Test the command that invokes the GeoMesa Tools:
+
+.. code-block:: bash
+
+    $ geomesa-kafka
+    Using GEOMESA_KAFKA_HOME = /path/to/geomesa-kafka-$KAFKAVERSION_2.11-$VERSION
+    Usage: geomesa-kafka [command] [command options]
+      Commands:
+        create          Create a feature definition in GeoMesa
+        describe        Describe the attributes of a given feature in GeoMesa
+        help            Show help
+        keywords        Add/Remove/List keywords on an existing schema
+        list            List GeoMesa features for a given zkPath
+        listen          Listen to a GeoMesa Kafka topic
+        removeschema    Remove a schema and associated features from GeoMesa
+        version         Display the installed GeoMesa version
+
+GeoMesa Tools comes bundled by default with an SLF4J implementation that is installed to the ``$GEOMESA_KAFKA_HOME/lib`` directory
```

Comment:
Since this SLF4J caveat is essentially identical for both the Accumulo and Kafka tools instructions, could we make this a separate section that both reference to?